src/Global/Safe.purs

```diff
@@ -0,0 +1,75 @@
+module Global.Safe
+  ( stringify
+  , toFixed
+  , toExponential
+  , toPrecision
+  , decodeURI
+  , encodeURI
+  , decodeURIComponent
+  , encodeURIComponent
+  ) where
+
+import Prelude
+import Data.Function.Uncurried (Fn3, Fn4, runFn3, runFn4)
+import Data.Maybe (Maybe(..))
+
+foreign import _stringify :: forall a b. Fn3 (String -> b) (String -> b) a b
+
+-- | Uses the global JSON object to turn anything into a string.
+-- | Returns `Nothing` when input is a function.
+stringify :: forall a. a -> Maybe String
+stringify v = runFn3 _stringify (const Nothing) Just v
```

Comment:
I would really rather not provide this function. The JSON libraries already have it, and they ensure the accepted type actually is JSON too. A "print a representation of anything" function is ok in `purescript-debug`, since you can't get a value out of it and all those functions raise a warning, but it's highly dodgy otherwise.